src/PrestaShopBundle/Form/Admin/Product/ProductSpecificPrice.php

```diff
@@ -164,7 +164,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'sp_id_customer',
                 TypeaheadCustomerCollectionType::class,
                 [
-                    'remote_url' => $this->context->getAdminLink('AdminCustomers', true) . '&sf2=1&ajax=1&tab=AdminCustomers&action=searchCustomers&customer_search=%QUERY',
+                    'remote_url' => $this->router->generate('admin_customers_search', ['sf2' => 1]) . '&customer_search=%QUERY',
```

Comment:
@matks i did fix it by appending query param. i know it's not the best solution but i dont think that having dedicated service just to provide url is any better. :/ basically what it does, it keeps `%` sign in URL.

if i were to use ` $this->router->generate('admin_customers_search', ['sf2' => 1, 'customer_search' => '%QUERY'])` it would generate `
/admin-dev/index.php/sell/customers/search?sf2=1&customer_search=%25QUERY`

another thing i could do is try to rework it to avoid using ` 'customer_search' => '%QUERY'` at all, wdyt?

Reply:
You're right, let's keep it like this for now. Please just add a warning statement (maybe a `@todo` ?).
For the `%` being encoded, if we encounter a 2nd time this issue, it'll mean we need to enable "raw params adding" for `getAdminLink()` function (because the `%` is converted in `%25` because of url encoding).

Reply:
added note about it.